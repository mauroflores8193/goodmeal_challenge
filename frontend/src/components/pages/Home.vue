<template>
  <div class="flex flex-col h-full">
    <Header title="Inicio"/>
    <div class="px-6 flex flex-col gap-4 flex-1 overflow-y-auto">
      <router-link :to="'/store/'+store.id" v-for="(store, index) in stores" :key="index">
        <StoreCard v-bind="store"/>
      </router-link>
    </div>
    <Nav/>
  </div>
</template>

<script>
import Nav from "../molecules/Nav.vue";
import StoreCard from "../organisms/StoreCard.vue";
import Header from "../molecules/Header.vue";

export default {
  name: 'Home',
  components: { Header, StoreCard, Nav },
  data: function () {
    return {
      stores: [
        {
          storeName: 'GoodMeal Market',
          banner: 'https://cdn.pixabay.com/photo/2017/03/22/12/14/easter-nest-2164822_960_720.jpg',
          attentionHours: 'Hoy 09:00 - 21:00 hrs',
          deliveryType: 'Retiro o delivery',
          icon: 'https://upload.wikimedia.org/wikipedia/commons/thumb/a/ab/Logo_TV_2015.png/200px-Logo_TV_2015.png',
          minProductPrice: 4500,
          minProductOldPrice: 9000,
          distanceTime: '45 min',
          distanceKm: '3.83 km',
          ordersCount: 11,
          isFavorite: true,
          id: 12345
        },
        {
          storeName: 'Natural King',
          banner: 'https://cdn.pixabay.com/photo/2017/07/23/16/01/nature-2531761__340.jpg',
          attentionHours: 'Hoy 09:00 - 21:00 hrs',
          deliveryType: 'Retiro o delivery',
          icon: 'https://www.shutterstock.com/image-vector/square-logo-design-vector-blue-260nw-706178086.jpg',
          minProductPrice: 4500,
          minProductOldPrice: 9000,
          distanceTime: '6 min',
          distanceKm: '0,51 km',
          ordersCount: 3,
          isFavorite: false,
          id: 12346
        }
      ]
    }
  }
}
</script>
