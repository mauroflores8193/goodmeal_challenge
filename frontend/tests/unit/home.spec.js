import { createLocalVue, mount } from '@vue/test-utils'
import VueRouter from 'vue-router'
import Home from '@/components/pages/Home.vue'
import flushPromises from 'flush-promises'
import axios from 'axios'
import storeList from './../mocks/storeList.json'

jest.mock('axios')

const localVue = createLocalVue()
localVue.use(VueRouter)
const router = new VueRouter()
test('increments value on click', async () => {
  axios.get.mockResolvedValue({ data: storeList });
  const tree = mount(Home, { localVue, router })
  await flushPromises();
  expect(tree).toMatchSnapshot();
})
