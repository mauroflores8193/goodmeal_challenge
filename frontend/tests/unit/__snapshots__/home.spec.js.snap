// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`increments value on click 1`] = `
<div class="flex flex-col h-full">
  <div class="h-16 flex relative items-center">
    <!---->
    <h1 class="text-lg font-medium mx-auto">Inicio</h1>
  </div>
  <div class="px-6 flex flex-col gap-4 flex-1 overflow-y-auto"><a href="#/store/1" class="">
      <div class="card" id="1" score="0.7" location="Fuga temporibus dicta temporibus expedita quia." latitude="-0.0600252813" longitude="-20.1813547873" products="[object Object],[object Object]">
        <div class="relative"><img src="https://cdn.pixabay.com/photo/2015/10/11/11/20/banner-982162__340.jpg" alt="banner" class="h-18 w-full rounded-t-xl"> <img src="https://99designs-blog.imgix.net/blog/wp-content/uploads/2018/10/attachment_57810318-e1539290902964.png?auto=format&amp;q=60&amp;fit=max&amp;w=930" alt="logo" class="rounded-full absolute -bottom-6 right-8 w-16 h-16 bg-white">
          <div class="absolute right-4 top-0">
            <!---->
            <!---->
          </div>
          <div class="bg-pink-500 text-white font-medium w-fit px-2 py-1 text-sm rounded-full absolute left-4 top-2">Hoy 13:34 - 20:34 hrs</div>
          <div class="bg-pink-100 text-pink-700 font-medium w-fit px-5 py-1 text-xs rounded-full absolute left-4 bottom-2">Retiro o delivery</div>
        </div>
        <div class="px-4 py-3">
          <div class="font-bold text-xl">Craig Terry</div>
          <div class="flex gap-2 mb-1">
            <div class="price">Desde 446.4</div>
            <div class="old-price">1488</div>
          </div>
          <div class="flex gap-6">
            <div class="flex items-center gap-1">
              <!---->
              <div>17 min</div>
            </div>
            <div class="flex items-center gap-1 flex-1">
              <!---->
              <div>5.7 km</div>
            </div>
            <div class="flex items-center gap-1">
              <div class="text-xl font-bold">3</div>
              <!---->
            </div>
          </div>
        </div>
      </div>
    </a><a href="#/store/2" class="">
      <div class="card" id="2" score="2" location="Placeat rerum eius repellat in in. Quis laborum quasi voluptatem in." latitude="48.1450550063" longitude="-111.9465288203" products="[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object]">
        <div class="relative"><img src="https://cdn.pixabay.com/photo/2016/03/06/17/39/banner-1240822__340.jpg" alt="banner" class="h-18 w-full rounded-t-xl"> <img src="https://www.clipartmax.com/png/middle/334-3341544_black-square-logo-square-point-of-sale-logo.png" alt="logo" class="rounded-full absolute -bottom-6 right-8 w-16 h-16 bg-white">
          <div class="absolute right-4 top-0">
            <!---->
            <!---->
          </div>
          <div class="bg-pink-500 text-white font-medium w-fit px-2 py-1 text-sm rounded-full absolute left-4 top-2">Hoy 15:58 - 23:58 hrs</div>
          <div class="bg-pink-100 text-pink-700 font-medium w-fit px-5 py-1 text-xs rounded-full absolute left-4 bottom-2">Retiro o delivery</div>
        </div>
        <div class="px-4 py-3">
          <div class="font-bold text-xl">Dr. Maxine Predovic DDS</div>
          <div class="flex gap-2 mb-1">
            <div class="price">Desde 149.2</div>
            <div class="old-price">1492</div>
          </div>
          <div class="flex gap-6">
            <div class="flex items-center gap-1">
              <!---->
              <div>13 min</div>
            </div>
            <div class="flex items-center gap-1 flex-1">
              <!---->
              <div>4.9 km</div>
            </div>
            <div class="flex items-center gap-1">
              <div class="text-xl font-bold">0</div>
              <!---->
            </div>
          </div>
        </div>
      </div>
    </a><a href="#/store/3" class="">
      <div class="card" id="3" score="3.5" location="Magni necessitatibus fugit commodi amet." latitude="20.9934629272" longitude="132.1989682653" products="[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object]">
        <div class="relative"><img src="https://cdn.pixabay.com/photo/2016/05/27/08/51/mobile-phone-1419275__340.jpg" alt="banner" class="h-18 w-full rounded-t-xl"> <img src="https://www.designevo.com/res/templates/thumb_small/shape-square-abstract-friend.png" alt="logo" class="rounded-full absolute -bottom-6 right-8 w-16 h-16 bg-white">
          <div class="absolute right-4 top-0">
            <!---->
            <!---->
          </div>
          <div class="bg-pink-500 text-white font-medium w-fit px-2 py-1 text-sm rounded-full absolute left-4 top-2">Hoy 06:26 - 14:26 hrs</div>
          <div class="bg-pink-100 text-pink-700 font-medium w-fit px-5 py-1 text-xs rounded-full absolute left-4 bottom-2">Retiro o delivery</div>
        </div>
        <div class="px-4 py-3">
          <div class="font-bold text-xl">Alessandra Hand</div>
          <div class="flex gap-2 mb-1">
            <div class="price">Desde 295.4</div>
            <div class="old-price">2954</div>
          </div>
          <div class="flex gap-6">
            <div class="flex items-center gap-1">
              <!---->
              <div>6 min</div>
            </div>
            <div class="flex items-center gap-1 flex-1">
              <!---->
              <div>5.7 km</div>
            </div>
            <div class="flex items-center gap-1">
              <div class="text-xl font-bold">0</div>
              <!---->
            </div>
          </div>
        </div>
      </div>
    </a><a href="#/store/4" class="">
      <div class="card" id="4" score="1.9" location="Rerum porro ducimus quia minus aut. Et vel aspernatur quibusdam et earum ex." latitude="-60.7210149293" longitude="91.4972851386" products="[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object]">
        <div class="relative"><img src="https://cdn.pixabay.com/photo/2015/12/15/09/04/banner-1093909__340.jpg" alt="banner" class="h-18 w-full rounded-t-xl"> <img src="https://www.creativefabrica.com/wp-content/uploads/2020/09/21/Modern-Square-Logo-Design-Graphics-5580883-1-580x386.jpg" alt="logo" class="rounded-full absolute -bottom-6 right-8 w-16 h-16 bg-white">
          <div class="absolute right-4 top-0">
            <!---->
            <!---->
          </div>
          <div class="bg-pink-500 text-white font-medium w-fit px-2 py-1 text-sm rounded-full absolute left-4 top-2">Hoy 20:17 - 03:17 hrs</div>
          <div class="bg-pink-100 text-pink-700 font-medium w-fit px-5 py-1 text-xs rounded-full absolute left-4 bottom-2">Retiro o delivery</div>
        </div>
        <div class="px-4 py-3">
          <div class="font-bold text-xl">Virgie Kilback</div>
          <div class="flex gap-2 mb-1">
            <div class="price">Desde 83.4</div>
            <div class="old-price">278</div>
          </div>
          <div class="flex gap-6">
            <div class="flex items-center gap-1">
              <!---->
              <div>41 min</div>
            </div>
            <div class="flex items-center gap-1 flex-1">
              <!---->
              <div>3.9 km</div>
            </div>
            <div class="flex items-center gap-1">
              <div class="text-xl font-bold">3</div>
              <!---->
            </div>
          </div>
        </div>
      </div>
    </a><a href="#/store/5" class="">
      <div class="card" id="5" score="2.5" location="Officiis quia perspiciatis nulla quia." latitude="27.2245772077" longitude="-16.2387477589" products="[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object]">
        <div class="relative"><img src="https://cdn.pixabay.com/photo/2018/06/07/09/01/emotions-3459666__340.jpg" alt="banner" class="h-18 w-full rounded-t-xl"> <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTp-bdZaY-8wXJXi_m4BjVC2L3new_7-_3WMA&amp;usqp=CAU" alt="logo" class="rounded-full absolute -bottom-6 right-8 w-16 h-16 bg-white">
          <div class="absolute right-4 top-0">
            <!---->
            <!---->
          </div>
          <div class="bg-pink-500 text-white font-medium w-fit px-2 py-1 text-sm rounded-full absolute left-4 top-2">Hoy 16:10 - 00:10 hrs</div>
          <div class="bg-pink-100 text-pink-700 font-medium w-fit px-5 py-1 text-xs rounded-full absolute left-4 bottom-2">Retiro o delivery</div>
        </div>
        <div class="px-4 py-3">
          <div class="font-bold text-xl">Chauncey Kihn</div>
          <div class="flex gap-2 mb-1">
            <div class="price">Desde 575.6</div>
            <div class="old-price">2878</div>
          </div>
          <div class="flex gap-6">
            <div class="flex items-center gap-1">
              <!---->
              <div>5 min</div>
            </div>
            <div class="flex items-center gap-1 flex-1">
              <!---->
              <div>0.8 km</div>
            </div>
            <div class="flex items-center gap-1">
              <div class="text-xl font-bold">3</div>
              <!---->
            </div>
          </div>
        </div>
      </div>
    </a><a href="#/store/6" class="">
      <div class="card" id="6" score="0.5" location="Necessitatibus natus aut aut sequi omnis. Cupiditate recusandae itaque ipsa ut est vel qui." latitude="48.3210479833" longitude="-30.7612855037" products="[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object]">
        <div class="relative"><img src="https://cdn.pixabay.com/photo/2017/10/03/17/53/nature-2813487__340.jpg" alt="banner" class="h-18 w-full rounded-t-xl"> <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTp-bdZaY-8wXJXi_m4BjVC2L3new_7-_3WMA&amp;usqp=CAU" alt="logo" class="rounded-full absolute -bottom-6 right-8 w-16 h-16 bg-white">
          <div class="absolute right-4 top-0">
            <!---->
            <!---->
          </div>
          <div class="bg-pink-500 text-white font-medium w-fit px-2 py-1 text-sm rounded-full absolute left-4 top-2">Hoy 01:14 - 10:14 hrs</div>
          <div class="bg-pink-100 text-pink-700 font-medium w-fit px-5 py-1 text-xs rounded-full absolute left-4 bottom-2">Retiro o delivery</div>
        </div>
        <div class="px-4 py-3">
          <div class="font-bold text-xl">Mr. Hank Schumm</div>
          <div class="flex gap-2 mb-1">
            <div class="price">Desde 84.6</div>
            <div class="old-price">141</div>
          </div>
          <div class="flex gap-6">
            <div class="flex items-center gap-1">
              <!---->
              <div>40 min</div>
            </div>
            <div class="flex items-center gap-1 flex-1">
              <!---->
              <div>5 km</div>
            </div>
            <div class="flex items-center gap-1">
              <div class="text-xl font-bold">1</div>
              <!---->
            </div>
          </div>
        </div>
      </div>
    </a><a href="#/store/7" class="">
      <div class="card" id="7" score="1.3" location="Optio voluptatem accusantium est voluptatem unde quo veritatis deleniti. Quae omnis rem doloribus autem fugit iste." latitude="-87.0045011756" longitude="28.5763816762" products="[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object]">
        <div class="relative"><img src="https://cdn.pixabay.com/photo/2016/09/04/20/14/sunset-1645103__340.jpg" alt="banner" class="h-18 w-full rounded-t-xl"> <img src="https://www.creativefabrica.com/wp-content/uploads/2020/09/21/Modern-Square-Logo-Design-Graphics-5580883-1-580x386.jpg" alt="logo" class="rounded-full absolute -bottom-6 right-8 w-16 h-16 bg-white">
          <div class="absolute right-4 top-0">
            <!---->
            <!---->
          </div>
          <div class="bg-pink-500 text-white font-medium w-fit px-2 py-1 text-sm rounded-full absolute left-4 top-2">Hoy 11:28 - 18:28 hrs</div>
          <div class="bg-pink-100 text-pink-700 font-medium w-fit px-5 py-1 text-xs rounded-full absolute left-4 bottom-2">Retiro o delivery</div>
        </div>
        <div class="px-4 py-3">
          <div class="font-bold text-xl">Kale Reilly</div>
          <div class="flex gap-2 mb-1">
            <div class="price">Desde 84</div>
            <div class="old-price">210</div>
          </div>
          <div class="flex gap-6">
            <div class="flex items-center gap-1">
              <!---->
              <div>3 min</div>
            </div>
            <div class="flex items-center gap-1 flex-1">
              <!---->
              <div>1.4 km</div>
            </div>
            <div class="flex items-center gap-1">
              <div class="text-xl font-bold">1</div>
              <!---->
            </div>
          </div>
        </div>
      </div>
    </a><a href="#/store/8" class="">
      <div class="card" id="8" score="1" location="Veritatis iusto sed voluptatem fugiat." latitude="-29.291639188" longitude="28.8025314588" products="[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object]">
        <div class="relative"><img src="https://cdn.pixabay.com/photo/2015/10/11/11/20/banner-982162__340.jpg" alt="banner" class="h-18 w-full rounded-t-xl"> <img src="https://www.clipartmax.com/png/middle/334-3341544_black-square-logo-square-point-of-sale-logo.png" alt="logo" class="rounded-full absolute -bottom-6 right-8 w-16 h-16 bg-white">
          <div class="absolute right-4 top-0">
            <!---->
            <!---->
          </div>
          <div class="bg-pink-500 text-white font-medium w-fit px-2 py-1 text-sm rounded-full absolute left-4 top-2">Hoy 13:02 - 21:02 hrs</div>
          <div class="bg-pink-100 text-pink-700 font-medium w-fit px-5 py-1 text-xs rounded-full absolute left-4 bottom-2">Retiro o delivery</div>
        </div>
        <div class="px-4 py-3">
          <div class="font-bold text-xl">Chadrick Kshlerin DVM</div>
          <div class="flex gap-2 mb-1">
            <div class="price">Desde 268.6</div>
            <div class="old-price">2686</div>
          </div>
          <div class="flex gap-6">
            <div class="flex items-center gap-1">
              <!---->
              <div>39 min</div>
            </div>
            <div class="flex items-center gap-1 flex-1">
              <!---->
              <div>5.1 km</div>
            </div>
            <div class="flex items-center gap-1">
              <div class="text-xl font-bold">0</div>
              <!---->
            </div>
          </div>
        </div>
      </div>
    </a><a href="#/store/9" class="">
      <div class="card" id="9" score="1.7" location="Consequatur fuga quibusdam voluptatibus eligendi mollitia." latitude="57.6778603753" longitude="-109.0744132032" products="[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object]">
        <div class="relative"><img src="https://cdn.pixabay.com/photo/2017/04/11/15/55/animals-2222007__340.jpg" alt="banner" class="h-18 w-full rounded-t-xl"> <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTS6hha1D-5eYlQibkJ-T0HxgsGQ_TfXqCT7w&amp;usqp=CAU" alt="logo" class="rounded-full absolute -bottom-6 right-8 w-16 h-16 bg-white">
          <div class="absolute right-4 top-0">
            <!---->
            <!---->
          </div>
          <div class="bg-pink-500 text-white font-medium w-fit px-2 py-1 text-sm rounded-full absolute left-4 top-2">Hoy 07:22 - 14:22 hrs</div>
          <div class="bg-pink-100 text-pink-700 font-medium w-fit px-5 py-1 text-xs rounded-full absolute left-4 bottom-2">Retiro o delivery</div>
        </div>
        <div class="px-4 py-3">
          <div class="font-bold text-xl">Damion Prosacco</div>
          <div class="flex gap-2 mb-1">
            <div class="price">Desde 511.5</div>
            <div class="old-price">1705</div>
          </div>
          <div class="flex gap-6">
            <div class="flex items-center gap-1">
              <!---->
              <div>34 min</div>
            </div>
            <div class="flex items-center gap-1 flex-1">
              <!---->
              <div>1.3 km</div>
            </div>
            <div class="flex items-center gap-1">
              <div class="text-xl font-bold">0</div>
              <!---->
            </div>
          </div>
        </div>
      </div>
    </a><a href="#/store/10" class="">
      <div class="card" id="10" score="2.3" location="Aliquam et et fugiat corrupti et est culpa." latitude="-42.9132582959" longitude="11.1865752336" products="[object Object]">
        <div class="relative"><img src="https://cdn.pixabay.com/photo/2016/05/27/08/51/mobile-phone-1419275__340.jpg" alt="banner" class="h-18 w-full rounded-t-xl"> <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTp-bdZaY-8wXJXi_m4BjVC2L3new_7-_3WMA&amp;usqp=CAU" alt="logo" class="rounded-full absolute -bottom-6 right-8 w-16 h-16 bg-white">
          <div class="absolute right-4 top-0">
            <!---->
            <!---->
          </div>
          <div class="bg-pink-500 text-white font-medium w-fit px-2 py-1 text-sm rounded-full absolute left-4 top-2">Hoy 22:58 - 08:58 hrs</div>
          <div class="bg-pink-100 text-pink-700 font-medium w-fit px-5 py-1 text-xs rounded-full absolute left-4 bottom-2">Retiro o delivery</div>
        </div>
        <div class="px-4 py-3">
          <div class="font-bold text-xl">Eulalia Crona</div>
          <div class="flex gap-2 mb-1">
            <div class="price">Desde 1049.6</div>
            <div class="old-price">5248</div>
          </div>
          <div class="flex gap-6">
            <div class="flex items-center gap-1">
              <!---->
              <div>17 min</div>
            </div>
            <div class="flex items-center gap-1 flex-1">
              <!---->
              <div>3.4 km</div>
            </div>
            <div class="flex items-center gap-1">
              <div class="text-xl font-bold">0</div>
              <!---->
            </div>
          </div>
        </div>
      </div>
    </a><a href="#/store/11" class="">
      <div class="card" id="11" score="2.2" location="Non officiis modi odit illum nostrum. Hic voluptas eaque accusamus voluptatum assumenda et." latitude="-40.4599941757" longitude="-83.0124552096" products="[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object]">
        <div class="relative"><img src="https://cdn.pixabay.com/photo/2016/09/04/20/14/sunset-1645105__340.jpg" alt="banner" class="h-18 w-full rounded-t-xl"> <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRF4mitZegM5djlD1bSaoI_klXEF4VNC2drlg&amp;usqp=CAU" alt="logo" class="rounded-full absolute -bottom-6 right-8 w-16 h-16 bg-white">
          <div class="absolute right-4 top-0">
            <!---->
            <!---->
          </div>
          <div class="bg-pink-500 text-white font-medium w-fit px-2 py-1 text-sm rounded-full absolute left-4 top-2">Hoy 08:42 - 16:42 hrs</div>
          <div class="bg-pink-100 text-pink-700 font-medium w-fit px-5 py-1 text-xs rounded-full absolute left-4 bottom-2">Retiro o delivery</div>
        </div>
        <div class="px-4 py-3">
          <div class="font-bold text-xl">Norbert Torp</div>
          <div class="flex gap-2 mb-1">
            <div class="price">Desde 214.8</div>
            <div class="old-price">358</div>
          </div>
          <div class="flex gap-6">
            <div class="flex items-center gap-1">
              <!---->
              <div>12 min</div>
            </div>
            <div class="flex items-center gap-1 flex-1">
              <!---->
              <div>5.9 km</div>
            </div>
            <div class="flex items-center gap-1">
              <div class="text-xl font-bold">2</div>
              <!---->
            </div>
          </div>
        </div>
      </div>
    </a><a href="#/store/12" class="">
      <div class="card" id="12" score="4.8" location="Saepe pariatur aut et corporis dicta. Ea consequatur temporibus ipsa ea hic et voluptas eveniet." latitude="-55.5354442473" longitude="70.5327148412" products="[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object]">
        <div class="relative"><img src="https://cdn.pixabay.com/photo/2016/09/04/20/14/sunset-1645103__340.jpg" alt="banner" class="h-18 w-full rounded-t-xl"> <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSMgMFQE7744Ebt1v172DZe85QHbKfeb7FHDg&amp;usqp=CAU" alt="logo" class="rounded-full absolute -bottom-6 right-8 w-16 h-16 bg-white">
          <div class="absolute right-4 top-0">
            <!---->
            <!---->
          </div>
          <div class="bg-pink-500 text-white font-medium w-fit px-2 py-1 text-sm rounded-full absolute left-4 top-2">Hoy 09:17 - 18:17 hrs</div>
          <div class="bg-pink-100 text-pink-700 font-medium w-fit px-5 py-1 text-xs rounded-full absolute left-4 bottom-2">Retiro o delivery</div>
        </div>
        <div class="px-4 py-3">
          <div class="font-bold text-xl">Miss Lilly Parisian</div>
          <div class="flex gap-2 mb-1">
            <div class="price">Desde 129.8</div>
            <div class="old-price">1298</div>
          </div>
          <div class="flex gap-6">
            <div class="flex items-center gap-1">
              <!---->
              <div>26 min</div>
            </div>
            <div class="flex items-center gap-1 flex-1">
              <!---->
              <div>4 km</div>
            </div>
            <div class="flex items-center gap-1">
              <div class="text-xl font-bold">0</div>
              <!---->
            </div>
          </div>
        </div>
      </div>
    </a></div>
  <nav class="flex h-20 justify-around items-center"><a href="#/" aria-current="page" class="router-link-exact-active router-link-active">
      <div class="button-icon text-pink-500">
        <!---->
        <!---->
        <div>Inicio</div>
      </div>
    </a> <a href="#/orders" class="">
      <div class="button-icon">
        <!---->
        <!---->
        <div>Órdenes</div>
      </div>
    </a> <a href="#/profile" class="">
      <div class="button-icon">
        <!---->
        <!---->
        <div>Perfil</div>
      </div>
    </a></nav>
</div>
`;
